app/views/courses/_info_window.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="card-course mt-2 mr-2">
```

Comment:
Possible to make the card clickable to access the show page of the course ?

Reply:
Done!